ignite/pkg/cosmosgen/generate_vuex.go

```diff
@@ -25,25 +25,41 @@ func newVuexGenerator(g *generator) *vuexGenerator {
 }
 
 func (g *generator) updateVueDependencies() error {
-	// Init the path to the "vue" folder inside the app
+	// Init the path to the "vue" and "vuex" folders inside the app
 	vuePath := filepath.Join(g.appPath, "vue")
 	packagesPath := filepath.Join(vuePath, "package.json")
+	vuexPackagesPath := filepath.Join(g.o.vuexRootPath, "package.json")
 	if _, err := os.Stat(packagesPath); errors.Is(err, os.ErrNotExist) {
 		return nil
 	}
 
+	if _, err := os.Stat(vuexPackagesPath); errors.Is(err, os.ErrNotExist) {
+		return nil
+	}
+
```

Comment:
Relative path to the TS client dependency won't be added if Vue is not scaffolded in the default location. It is not strictly an error but might confuse users during development. It could happen if a user add the `--path` flag to the `scaffold vue` command.

I think we should take some time to think on the user workflow when scaffolding/generating frontend code. Maybe in a different issue.

Reply:
wdym? aren't relative paths calculated according to the generated ts-client path and the path in question?

Reply:
I mean that if user runs `generate vuex` and there is no `./vue` folder because `scaffold vue` was not run it will silently fail to link to the `ts-client` package to Vuex's `packages.json`. The exists check for `packagesPath` in line 32 would fail returning `nil`.

Remember that we won't scaffold Vue by default, so users would need to run `scaffold vue` themselves.

Reply:
ah...I see what you mean...

Split it into 2 separate functions so lack of an app if only generating the stores won't interfere with settign up their dependencies.